toolchain/driver/testdata/error_codegen_target_triple.carbon

```diff
@@ -0,0 +1,8 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// RUN:  %{not} %{carbon} dump assembly --target_triple=x86_687-unknown-linux-gnu %s | %{FileCheck-strict}
+// CHECK:STDERR: {{ERROR:.*}}
```

Comment:
```suggestion
// CHECK:STDERR: ERROR: {{.*}}
```

Let's limit the regex to only be used where needed. Also, there should be a space after "ERROR:", so you can validate that.

Reply:
I see you removed the space after ERROR:. Why shouldn't a space be there?